compiler-core/src/erlang/tests/snapshots/gleam_core__erlang__tests__type_params__custom_type_named_args_count_once.snap.new

```diff
@@ -0,0 +1,20 @@
+---
+source: compiler-core/src/erlang/tests/type_params.rs
+assertion_line: 38
+expression: "\n        pub type Wibble(a, b) {\n            Wibble(a, b)\n        }\n\n        pub fn wibble() -> Wibble(a, a) {\n            todo\n        }\n        "
+---
+-module(my@mod).
+-compile([no_auto_import, nowarn_unused_vars, nowarn_unused_function, nowarn_nomatch]).
+
+-export([wibble/0]).
+-export_type([wibble/2]).
+
+-type wibble(I, J) :: {wibble, I, J}.
+
+-spec wibble() -> wibble(any(), any()).
```

Comment:
Shouldn't this be `wibble(a, a)`?

Reply:
yes, that’s still a unaccepted snapshot i think

Reply:
Will try to tackle this PR tonight, I think I have the solution: counting of variables should be done by taking max of count for each constructor. Will get back to you if this works out.